conserver/README

```diff
@@ -4,28 +4,28 @@ To make the conserver-xcat rpm, run:
 
 To update the initscript.patch or initscript1.patch:
```

Comment:
Hi, @chenglch , the pull request looks good to me, will you pls add more change log here such as initscript.patch and initscript1.patch will be replaced by initscript_8.2.1.patch....